virttest/qemu_vm.py

```diff
@@ -5360,15 +5360,19 @@ def eject_cdrom(self, device, force=False):
         else:
             return self.monitor.eject_cdrom(device, force)
 
-    def change_media(self, device, target):
+    def change_media(self, device, target, force=True):
```

Comment:
```suggestion
    def change_media(self, device, target, force=None):
```
This method doesn't work on the version in `( , 7.1.0) `if we set a valid value to `force` since the qemu version is older than version `7.1.0` which doesn't support the `force` option.
So it doesn't make sense in this situation.
And we should keep the previous cases' behaviors that don't set the force option.

Reply:
updated